swap_meet/vendor.py

```diff
@@ -1,2 +1,82 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory
+        if self.inventory == None:
+            self.inventory = []
```

Comment:
Nice job avoiding a mutable default argument! One small thing, remember to do `is None` instead of `== None` when checking whether an object is `None`.